rpcs3/Emu/RSX/Common/expected.hpp

```diff
@@ -94,11 +94,11 @@ namespace rsx
             return value;
         }
 
-        template<typename = std::enable_if<!std::is_same_v<T, bool>>>
-        operator bool() const
-        {
-            return error.empty();
-        }
+		operator bool() const
```

Comment:
Indentation

Reply:
The indentation is correct now, it was using spaces before. Some lines in the file use spaces others tabs.